### ISSUE TITLE
Can't like a post after 1st signing up

### ISSUE THREAD

**Issue by joshmorel** [FightPandemics/FightPandemics#1090]
@kaydeejay I think this issue is related to #923 

This endpoint doesn't work after signing up because the POST users/ returns `_id` as `_id` not `id` 

```
const endPoint = `/api/posts/${postId}/likes/${user && user.id}`;
```

I wish we had always sent back `_id` as `_id` instead of sometimes `id` but for now can you fix this one?

**Comment by kaydeejay** [comment]
Sure thing, @joshmorel ! Happy to take a look at it.

**Comment by idjevm** [comment]
@joshmorel so if I refresh it works, doesn't it have to do with state or something like that?

**Comment by joshmorel** [comment]
@idjevm No it's because of the data sent from the backend. After creation the user object has _id where as get current user (called on refresh) it's id

Renaming _id to id in some routes was a mistake imho

**Comment by idjevm** [comment]
aaaaht that makes sense. :O Damn hopefully we can get this done soon @kaydeejay, let us know if you have any questions

**Comment by kaydeejay** [comment]
Will do @idjevm , I plan to spend a good of time on this tomorrow and Sunday morning (EST). I'll reach out to you or @joshmorel if I have any questions. Thanks!

**Comment by joshmorel** [comment]
Resolved by #1136
